templates/zerver/accounts_send_confirm.html

```diff
@@ -1,6 +1,10 @@
 {% extends "zerver/portico_signup.html" %}
 {# Displayed after a user attempts to sign up. #}
 
+{% block title %}
+<title>{{ _("Zulip: Confirm sign up") }}</title>
```

Comment:
Should be “Zulip: Confirm signup” (“sign up” is a noun; “signup” is a verb), or more readably “Zulip: Confirm your email address”.

Reply:
Thanks for catching that. Going with the more readable option, "Zulip: Confirm your email address".